src/components/TaskTracker/Task.tsx

```diff
@@ -66,7 +66,7 @@ export const Task = ({ task, tasks }) => {
   };
 
   const markNotCompleteWhenTracking = () => {
-    if (!task.inProgress) toggleInProgressState(task.id);
+    toggleInProgressState(task.id);
```

Comment:
@justnat3 IIRC you created this implementation - does this look OK to you/

Reply:
@Sakelig, I seem to remember this bug... Honestly thought this was fixed- 

Your patch does fix the behavior, thank you! I do have one nit-pick. While you are in there could you change the text dependent on the state to "track"/"un-track" - and "complete"/"not-finished"

Cheers,
-Nate